src/App.tsx

```diff
@@ -1,24 +1,157 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from "react";
+import { UserWarning } from "./UserWarning";
+import { getTodos, postTodo, deleteTodo } from "./api/todos";
+import { TodoInterface } from "./types/todo";
```

Comment:
Can't see the todo file
<img width="567" alt="image" src="https://user-images.githubusercontent.com/99403198/232441299-b19cfd47-378f-493a-bb0b-1ed34005cdfc.png">
Try this 
```suggestion
import { TodoInterface } from "./types/Todo";
```